raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -825,6 +848,42 @@ describe('Raiden', () => {
     expect(mockedTransfer.mock.calls[1]).toEqual([partner, BigNumber.from(1_000_000)]);
   });
 
+  test('closeChannel', async () => {
+    const closeBlock = 60;
+    function channelCloseEpicMock(action$: Observable<RaidenAction>) {
+      return action$.pipe(
+        filter(channelClose.request.is),
+        mapTo(
+          channelClose.success(
+            {
+              id: channelId,
+              participant: address,
+              txHash,
+              txBlock: closeBlock,
+              confirmed: true,
+            },
+            { tokenNetwork, partner },
+          ),
+        ),
+      );
+    }
+    const deps = makeDummyDependencies();
+    deps.registryContract = {
+      token_to_token_networks: jest.fn(async () => tokenNetwork),
+    } as any;
+    const raiden = new Raiden(
+      makeInitialState(
+        { address, network, contractsInfo },
+        { tokens: { [token]: tokenNetwork }, channels: { [key]: getChannel() } },
+      ),
+      deps,
+      combineRaidenEpics([initEpicMock, channelCloseEpicMock]),
+      dummyReducer,
+    );
+    await expect(raiden.start()).resolves.toBeUndefined();
+    await expect(raiden.closeChannel(token, partner, { subkey: false })).resolves.toEqual(txHash);
```

Comment:
Please, also verify that a `promise` with `channelClose.request` action got emitted with the desired payload

Reply:
Ok will do